internal/tools/tool/ads.go

```diff
@@ -5,63 +5,31 @@ package tool
 
 import (
 	"github.com/microsoft/go-sqlcmd/internal/io/file"
-	"os"
-	"path/filepath"
+	"github.com/microsoft/go-sqlcmd/internal/test"
 )
 
 type AzureDataStudio struct {
-	Base
+	tool
 }
 
 func (t *AzureDataStudio) Init() {
-	t.Base.SetName("ads")
-	userProfile := os.Getenv("USERPROFILE")
-	programFiles := os.Getenv("ProgramFiles")
+	t.tool.SetToolDescription(Description{
+		Name:        "ads",
+		Purpose:     "Azure Data Studio provides a user interface for working with SQL Server and Azure SQL Database.",
```

Comment:
>"Azure Data Studio provides a user interface for working with SQL Server and Azure SQL Database [](http://example.com/codeflow?start=15&length=95)

did Erin sign off on this description?

Reply:
Nope.

Reply:
Changed the same text used here:

[https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall](https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall)

Purpose:     "Azure Data Studio is a database tool for data professionals who use on-premises and cloud data platforms.",